src/lib/homestore_backend/replication_state_machine.hpp

```diff
@@ -12,14 +12,57 @@ using homestore::repl_req_ctx;
 using homestore::ReplServiceError;
 
 struct ho_repl_ctx : public homestore::repl_req_ctx {
-    ReplicationMessageHeader header_;
     sisl::io_blob_safe hdr_buf_;
+    sisl::io_blob_safe key_buf_;
```

Comment:
comment will be helpful that data_bufs_ are kept so data_sgs_ buffers dont get destroyed.